### ISSUE TITLE
Plugin cannot be installed in  grafana running on arm64

### ISSUE THREAD

**Issue by sudhirpandey** [grafana/athena-datasource#94]
**What happened**:
Trying to install this plugin via cli in  grafana arm64 image errors out
grafana-cli --pluginsDir "/var/lib/grafana/plugins" plugins install grafana-athena-datasource
Error:  grafana-athena-datasource v is not supported on your system (Grafana v8.2.5 linux-arm64)

**What you expected to happen**:
Expect it to be installed like in amd64 grafana in image.

**How to reproduce it (as minimally and precisely as possible)**:
try installing the plugin in grafana that runs in gravition 

grafana-cli --pluginsDir "/var/lib/grafana/plugins" plugins install grafana-athena-datasource

**Screenshots**

**Anything else we need to know?**:

**Environment**:
- Grafana version: v8.2.5
- Plugin version: v0.2.3
- OS Grafana is installed on:  linux arm64

**Comment by andresmgot** [comment]
Thanks for filling this issue @sudhirpandey. Does it work for you if you download and unzip the arm64 zip file from the releases page? https://github.com/grafana/athena-datasource/releases

**Comment by sudhirpandey** [comment]
Managed to get it installed using this though

grafana-cli --pluginUrl https://github.com/grafana/athena-datasource/releases/download/v0.2.3/grafana-athena-datasource-0.2.3.linux_arm64.zip plugins install grafana-athena-datasource

**Comment by andresmgot** [comment]
That may be a bug in `grafana-cli`. I don't have an ARM environment to test this so do you mind trying a couple of things?

 - We just released 1.0.0 (which has the same content than 0.2.3). Can you install that version? If `grafana-cli --pluginsDir "/var/lib/grafana/plugins" plugins install grafana-athena-datasource` doesn't work, I would try specifying the verison: `grafana-cli --pluginsDir "/var/lib/grafana/plugins" plugins install grafana-athena-datasource 1.0.0`.
 - Can you install any other plugin?

Thanks in advance!

**Comment by iwysiu** [comment]
This issue is being closed due to lack of activity.

**Comment by fridgepoet** [comment]
I’ve run into this issue on my ARM machine -- observed the same behavior
* not supported with grafana-athena-datasource 1.0.0
* not supported with for example grafana-redshift-datasource either
* but grafana-azure-data-explorer-datasource was installed successfully

**Comment by andresmgot** [comment]
> * not supported with grafana-athena-datasource 1.0.0

just to confirm, it does affect all the versions, doesn't it?

**Comment by fridgepoet** [comment]
> just to confirm, it does affect all the versions, doesn't it?

Right, I get the message that it is not supported no matter the targeted version.

**Comment by miettal** [comment]
in 1.0.1, same problem happened.

this is actual log on AWS arm instance.
```
sudo /usr/sbin/grafana-cli plugins install grafana-athena-datasource 1.0.1
Error: ✗ grafana-athena-datasource v1.0.1 is not supported on your system (Grafana v8.3.4 linux-arm64)
```

**Comment by andresmgot** [comment]
There was an issue with the release process that prevented the arm64 binaries from being uploaded to Grafana. This issue is already fixed so the next release (1.0.2), which is on its way, will be properly released. You can still install 1.0.1 using the --pluginUrl flag:

```
grafana-cli --pluginUrl https://github.com/grafana/athena-datasource/releases/download/v1.0.1/grafana-athena-datasource-1.0.1.linux_arm64.zip plugins install grafana-athena-datasource
```

**Comment by andresmgot** [comment]
The version 1.0.2 has just been released and the arm64 binaries are there so this should be fixed! Let us know (and reopen) if it's not the case. Thanks!
